lib/core/api.js

```diff
@@ -69,6 +74,10 @@ const api = {
     pointers: ['const char*'],
     rpointer: 'DIR*'
   },
+  mkdir: {
```

Comment:
how would you define the `-p` flag?

Reply:
we will have to write the code to do that in js, which would be quite a bit simpler than the [equivalent](https://github.com/mirror/busybox/blob/master/libbb/make_directory.c#L27) C.